inbm/dispatcher-agent/dispatcher/schedule/sqlite_manager.py

```diff
@@ -24,8 +25,12 @@ def __init__(self, db_file=UDM_DB_FILE) -> None:
         """
         self._db_file = db_file
         if self._db_file != ":memory:" and not os.path.exists(self._db_file):
-            raise DispatcherException(f"Error: Database file {self._db_file} does not exist.")
-
+            # Create database file if not exist
+            logger.info(f"Database file doesn't exist. Creating the file.")
```

Comment:
<h3>$\color{#FFA500}Improvements: Code Maintainability$</h3>

When creating the database file if it does not exist, it's crucial to also initialize the database schema. Currently, the file is created, but tables are not created until an operation is attempted. Consider moving the schema creation into this conditional block to ensure the database is fully initialized from the start.